src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/Models/FrameworkDependencyModel.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+using Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Snapshot;
+using Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Subscriptions.RuleHandlers;
+
+#nullable enable
```

Comment:
Nit: This is now not required

Reply:
Fixed. There was one in my PR from earlier today as well which I've removed here.